pyQuARC/code/base_validator.py

```diff
@@ -38,3 +43,7 @@ def is_in(value, list_of_values):
     def compare(first, second, relation):
         func = getattr(BaseValidator, relation)
         return func(first, second)
+
+    @staticmethod
+    def cmr_request(cmr_prms):
+        return requests.get(f'{CMR_URL}/search/{cmr_prms}')
```

Comment:
```suggestion
        return requests.get(f'{CMR_URL}/search/{cmr_prms}')
```
Recommend including a new line at the end of the file. I think this is a best practice?